README.adoc

```diff
@@ -150,10 +147,18 @@ only supported orchestrators are reported.
   * `stop`: Stop the given service in the box.
   * `restart`: Restart the given service in the box.
 +
---
 The supported options for `service` are: Docker, OpenShift.
 
+1. `vagrant service-manager install-cli [service] [--cli-version] [--path]`
++
+Install the client binary for the specified `service`.
++
+The possible options are:
 
+  * `--cli-version`: Download the binary in the specified version, if it exists.
+  * `--path`: Download the binary to the specified path on the host.
++
+The supported options for `service` are: Docker, OpenShift.
```

Comment:
@budhrg : I saw that, the message line after successful command execution prints the path where binary is downloaded, and it has to be there for completeness. I mentioned about docs since it makes sense to note down such kind of information in documentation as well.